modules/python/src2/cv2.cpp

```diff
@@ -290,6 +335,64 @@ bool parseNumpyScalar(PyObject* obj, T& value)
     return false;
 }
 
+#ifdef CV_CXX11
+thread_local
+#endif
```

Comment:
C++98 code without TLS is not thread-safe and may lead to crashes.

Consider using OpenCV wrapper through `#include <opencv2/core/utils/tls.hpp>` instead.